src/components/Nav/MenuItem.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 import styled from 'styled-components/macro';
 
 function MenuItem({category}) {
     return (
         <li>
-            <MenuItemA>{category.title}</MenuItemA>
+            <MenuItemA>
+                <Link to={'/category/' + category.slug}>{category.title}</Link>
+            </MenuItemA>
```

Comment:
A `Link` from react-router is basically an anchor element. What you are doing here is nesting an `a` tag in an `a` tag. This is wrong. To get the styling, you need to style the `Link`.
```suggestion
                <Link to={'/category/' + category.slug}>{category.title}</Link>
```